WORKSPACE

```diff
@@ -33,5 +33,11 @@ load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
 go_rules_dependencies()
 go_register_toolchains()
 
+load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps", "grpc_test_only_deps")
```

Comment:
And why is this needed if it wasn't needed before? Just curiosity.

Reply:
Actually something more than curiosity: the build failures in Win/Mac are found in a dependency of grpc, probably loaded here.

Reply:
I had to add these to resolve grpc dependency issues.

See: https://github.com/grpc/grpc/blob/560577371abff8f504091d906ddaf58126a07762/bazel/grpc_extra_deps.bzl#L10-L27

But I guess we should be able to remove the `grpc_test_only_deps`, can you try this?

Reply:
I checked it locally and it works. The build seems to finish successfully without the test_only_deps

Reply:
However, I have only a linux machine where I can build. For some reason, bazel doesn't run on my Mac, so I'm not able to verify on any other platform.

Reply:
@silence-do-good thanks for checking, I removed the `grpc_test_only_deps`

Reply:
@pau-baiget I also added the link above as a comment.